camera/camera_manager.cc

```diff
@@ -16,68 +50,41 @@ CameraManager::CameraManager() {
 }
 
 void CameraManager::load_configs() {
-  // directory iteration from https://stackoverflow.com/questions/612097/how-can-i-get-the-list-of-files-in-a-directory-using-c-or-c/37494654#37494654
+  // directory iteration from
+  // https://stackoverflow.com/questions/612097/how-can-i-get-the-list-of-files-in-a-directory-using-c-or-c/37494654#37494654
   const std::string config_dir = "/jet/camera/cfg";
-  for (const auto & entry : fs::directory_iterator(config_dir)) {
+  for (const auto& entry : fs::directory_iterator(config_dir)) {
     const std::string filename = entry.path();
-    if (filename == "." || filename == ".."){ 
+    if (filename == "." || filename == "..") {
       continue;
     }
-    std::optional<YAML::Node> config = read_YAML(filename);
-    std::optional<Camera> camera = parse_config(*config);
+
+    const std::optional<YAML::Node> config = read_YAML(filename);
+    if (!config) {
+      throw std::runtime_error("Could not open config");
```

Comment:
Doesn't this throw if any file in the directory is not valid YAML?  Is that what we want, or do we just want to ignore these?

Reply:
Good pt